package.json

```diff
@@ -119,21 +118,22 @@
     "@grafana/data": "^11.3.0",
     "@grafana/lezer-logql": "^0.2.7",
     "@grafana/runtime": "^11.3.0",
-    "@grafana/scenes": "5.41.1",
+    "@grafana/scenes": "6.6.3",
     "@grafana/ui": "^11.3.0",
     "@hello-pangea/dnd": "^16.6.0",
     "@lezer/common": "^1.2.1",
     "@lezer/lr": "^1.4.1",
     "@types/react-table": "^7.7.20",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-router-dom": "^6.22.0",
     "rxjs": "7.8.1",
     "tslib": "2.5.3",
     "uuid": "^10.0.0"
   },
   "resolutions": {
-    "**/fast-loops": "^1.1.4"
```

Comment:
I think I accidentally removed this. I get no errors. The initial reason for this may no longer be relevant?

Reply:
@L2D2Grafana did we ever determine out if this re-introduces the security vuln that this change was intended to fix?

Reply:
Seems fine to remove, `react-use` took out fast-loops
https://github.com/robinweser/inline-style-prefixer/commit/4c1ffce41c08ebbc5e4e773f8de7149944270f1a
![Screenshot 2025-04-17 at 11 18 55 AM](https://github.com/user-attachments/assets/bc4a26ec-9754-4f39-a890-d849352e47e0) I can't find it in our code base.